### ISSUE TITLE
Bisection is not descending into V8 rolls for runtime callstats

### ISSUE THREAD

**Issue by natorion** [catapult-project/catapult#3204]
Example alerts: https://chromeperf.appspot.com/group_report?bug_id=690837

Relevant output:
==============
2017-02-10 10:53:30 
=== BISECT JOB RESULTS ===
Bisect was unable to run to completion

The bisect was able to narrow the range, you can try running with:
  good_revision: dd211c21b09bcba813a3c8c8bddff930e44f69a8
  bad_revision : bcd1a031bcd2e935901d83520339bcb089b57496

==============

Interesting other fact: The issue is not updated on Monorail with the result and the bisection status still says started: https://chromeperf.appspot.com/buildbucket_job_status/8988053272142074880

**Comment by natorion** [comment]
cc @camillobruni

**Comment by anniesullie** [comment]
@simonhatch can you take a look?

**Comment by anniesullie** [comment]
At first glance, it looks to me like the bisect is still running? @simonhatch is the bug that the quicklog was updated when the bisect is in progress?

**Comment by simonhatch** [comment]
Strange, that is a completed message but I think /update_bug_with_result is saying it isn't complete. Lemme add some logging and dig into this.

**Comment by simonhatch** [comment]
Actually I think Annie was correct, it's a partial result and the final result hadn't streamed in yet. Checked the logs and the last /post_bisect_results was at 10:18:28, I ran /update_bug_with_results with extra logging at 10:18:48 and it updated the bug properly with the results. I think maybe what we can do here is make it clear that this bisect is still in progress in the quicklog?

If we changed the header to read?

=== BISECT JOB RESULTS ===
Bisect is still in progress, results below are incomplete.
